GoodDollar/GoodDAPP
# (Bug) #1091 start claiming feed card is added multiple times

# Description

- patch .then() of gun, so it waits 200msec before gun.on returns. this solves the issue of empty node
- initialize feed in userstorage and await for its initial data
- start from events 24hours ago from last seend block on every wallet load so we dont miss events


# How to test
- on goodqa/ewallet close all incognito tabs
- open a new incognito tab
- paste a magiclink of a goodqa/ewallet user
result: user is not logged in 
expected result: user is logged in
- refresh -> user will be logged in
- look at feed
result: usually a duplicate card will be added (hanuka/start claiming)
expected result: no new system feed cards

This  should not be reproduced on this PR branch
